### PR TITLE
Add support for `BENTLEY_materials_point_style`

### LINKED ISSUE #1862: Support `BENTLEY_materials_point_style`
### Feature

[`BENTLEY_materials_point_style`](https://github.com/CesiumGS/glTF/tree/vendor-extensions/extensions/2.0/Vendor/BENTLEY_materials_point_style) is a lightweight extension for rendering points as circles with a specified pixel diameter.

We have something similar with point attenuation, where points are rendered as quads with a user-influenced pixel size. This could use the same rendering pipeline, just with an override for the desired pixel size + some shader code to render the points as circles.

### References

- [`BENTLEY_materials_point_style` spec](https://github.com/CesiumGS/glTF/tree/vendor-extensions/extensions/2.0/Vendor/BENTLEY_materials_point_style)
- CesiumJS PR: 
https://github.com/CesiumGS/cesium/pull/13093/changes

### PR DESCRIPTION
**Pull request by j9liu**
## Description

Depends on https://github.com/CesiumGS/cesium-native/pull/1408, so merge that first.

This PR adds support for the `BENTLEY_materials_point_style` extension, reusing the point attenuation pipeline with some minor adjustments.

| CesiumJS | Unreal |
| - | - |
| Image | image |

If this extension exists on a material used by the point primitive, then the other Point Cloud Shading parameters will have no effect on it. I can't tell what interactions would make sense from a user perspective here, but for now let's treat them as separate until there are use cases that require otherwise.

## Issue number or link

Closes #1862.

## Author checklist

- [X] I have submitted a [Contributor License Agreement](https://github.com/CesiumGS/community/tree/main/CLAs) (only needed once).
- [x] I have done a full self-review of my code.
- [X] I have updated `CHANGES.md` with a short summary of my change (for user-facing changes).
- [ ] I have added or updated unit tests to ensure consistent code coverage as necessary.
- [x] I have updated the documentation as necessary.

## Testing plan

Load [PointStyle.zip](https://github.com/user-attachments/files/29811976/PointStyle.zip) under a **CesiumGeoreference** with True Origin.

There should be four dots, and in order of smallest to largest: Red, Green, Yellow, Blue.

Also open `11_CesiumPointCloud` and verify that attenuation works as normal for those samples.

## Reviewer checklist

Thank you for taking the time to review this PR. By approving a PR you are taking as much responsibility for these changes as the author.

As you review, please go through the checklist below:

- [ ] Review and run all parts of the test plan on this branch and verify it matches expectations.
    - [ ] If the issue is a bug please make sure you can reproduce the bug in the main branch and then checkout this branch to make sure it actually solved the issue.
- [ ] Review the code and make sure you do not have any remaining questions or concerns. You should understand the code change and the chosen approach. If you are not confident or have doubts about the code, please do not hesitate to ask questions.
    - [ ] Code should follow the [C++ Style Guide](https://cesium.com/learn/cesium-native/ref-doc/style-guide.html)
- [ ] Review the unit tests and make sure there are no missing tests or edge cases.
- [ ] Review documentation changes and updates to `CHANGES.md` to make sure they accurately cover the work in this PR.
- [ ] Verify that the [Contributor License Agreement](https://github.com/CesiumGS/community/tree/main/CLAs) has been submitted, if needed.

### CODE REVIEW

**Comment by j9liu** [comment]
This is ready for review!

As I briefly stated in the description, this implementation always causes the points to render as circles with their specified diameter. I have ignored any interaction with the existing attenuation settings. Given that attenuation intends to enhance the appearance of regular point clouds, I think it stands to reason that points containing `BENTLEY_materials_point_style` already do not need this additional enhancement.

File: Shaders/Private/CesiumPointAttenuationVertexFactory.ush [hunk 1]
```diff
@@ -85,8 +86,9 @@ struct FVertexFactoryIntermediates
 struct FVertexFactoryInterpolantsVSToPS
 {
   	TANGENTTOWORLD_INTERPOLATOR_BLOCK
-  	half4  Color : COLOR0;
-
+  	half4   Color : COLOR0;
+  	float2  CornerOffset : CORNER_OFFSET;
+
```
Comment (timoore) [inline Shaders/Private/CesiumPointAttenuationVertexFactory.ush]: Extreme nitpick, but there are spaces on this line, and on other new lines in this file.

**Review by timoore — COMMENTED** [review]
Despite the fairly large amount of code, I don't have criticisms, so thank you very much @j9liu ! Could you fix the formatting problem in the shader code and update the cesium-native commit to `main`?

**Comment by j9liu** [comment]
Addressed your comments, thanks @timoore !
